packages/bot-web-ui/src/components/dashboard/bot-stop-notification.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useRef } from 'react';
+import { observer } from 'mobx-react';
+import { Icon, Toast } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { useDBotStore } from 'Stores/useDBotStore';
+
+const BotStopNotification = observer(() => {
+    const { run_panel } = useDBotStore();
+    const { setShowBotStopMessage } = run_panel;
+    const notification_timer = useRef(6000);
```

Comment:
i dont understand the reason that you need useRef for this static value, would you please give me some info?